### DISCUSSION TITLE
Vercel Vite blocking Clerk Cors

### DISCUSSION THREAD

**Discussion by SniperNoSnipey8** [vercel/community#3679] [Help]
### Summary

When trying to access my page https://app.investorradar.io clerk is blocked by CORS. This means I cant login or do anything. Clearing the cookies cache has helped but then it just restarts the error again. You can check it in the console.

> Access to script at 'https://clerk.investorradar.io/npm/@clerk/clerk-js@4/dist/clerk.browser.js' from origin 'https://app.investorradar.io' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
index.961aad5f.js:66     GET https://clerk.investorradar.io/npm/@clerk/clerk-js@4/dist/clerk.browser.js net::ERR_FAILED 404

### Example

https://app.investorradar.io

### Steps to Reproduce

Just login if it doesnt occur right away and then sign out and wait a little. It should happen within a few minutes.

**Comment by dgsunesen** [comment]
It seems you got it fixed? I'm having the exact same problem. 

What did you do to fix it?
